### PR TITLE
Export radToDeg method from NorthArrow to fix canvasSaveButton example

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

**Comment by friedjoff** [comment]
How about moving `radToDeg` to utils? I can imagine other places where this function might be useful. The current implementation makes it pretty hidden and easy to brake in the future.

**Comment by oterral** [comment]
Maybe use a existing very small npm module for this. so we don't have to care about this anymore.

File: src/components/NorthArrow/NorthArrow.js [hunk 1]
```diff
@@ -71,4 +70,6 @@ function NorthArrow({ map, rotationOffset, circled, children, ...other }) {
 NorthArrow.propTypes = propTypes;
 NorthArrow.defaultProps = defaultProps;
 
-export default React.memo(NorthArrow);
+const memoized = React.memo(NorthArrow);
+
+export default memoized;
```
Comment (oterral) [inline src/components/NorthArrow/NorthArrow.js]: go back to the oald version

**Comment by oterral** [comment]
@edellucien rebase fix then yiou can merge.

**Pull request merged**